docs/test-specifications/schedule-service/ScheduleSignTransaction.md

```diff
@@ -0,0 +1,119 @@
+| 6       | Sign a schedule that has already been executed  | scheduleId=<EXECUTED_SCHEDULE_ID>, commonTransactionParams.signers=[<VALID_SIGNER_KEY>] | Fails with `SCHEDULE_ALREADY_EXECUTED`.                                                                                                                  | N                 |
+
+---
+
+### **Signature Requirements**
+- Tests for different signature requirements and scenarios.
+
+| Test no | Name                                                                | Input                                                                                   | Expected Response                                                                                                                                                   | Implemented (Y/N) |
+|---------|---------------------------------------------------------------------|-----------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Sign a schedule with unauthorized signer                            | scheduleId=<VALID_SCHEDULE_ID>, commonTransactionParams.signers=[<UNAUTHORIZED_KEY>]    | Fails with `NO_NEW_VALID_SIGNATURES`. **Verification**: ScheduleInfo signatories array unchanged.                                                                   | N                 |
+| 2       | Sign a schedule with invalid signature                              | scheduleId=<VALID_SCHEDULE_ID>, commonTransactionParams.signers=[<INVALID_KEY>]         | Fails with `SOME_SIGNATURES_WERE_INVALID`. **Verification**: ScheduleInfo signatories array unchanged.                                                              | N                 |
+| 3       | Sign a schedule without providing any signatures                    | scheduleId=<VALID_SCHEDULE_ID>                                                          | Fails with `NO_NEW_VALID_SIGNATURES`. **Verification**: ScheduleInfo signatories array unchanged.                                                                   | N                 |
+| 4       | Sign a schedule that requires multiple signatures (first signature) | scheduleId=<MULTI_SIG_SCHEDULE_ID>, commonTransactionParams.signers=[<SIGNER_KEY_1>]    | Transaction succeeds, signature added to schedule. **Verification**: ScheduleInfo signatories array contains new entry; executed_timestamp remains null.            | N                 |
+| 5       | Sign a schedule that requires multiple signatures (all signatures)  | scheduleId=<MULTI_SIG_SCHEDULE_ID>, commonTransactionParams.signers=[<SIGNER_KEY_2>]    | Transaction succeeds, schedule executed immediately. **Verification**: ScheduleInfo executed_timestamp is set; signatories array contains both expected entries.    | N                 |
```

Comment:
Does this need `<SIGNER_KEY_1>` in the `signers` list as well? The test name specifies all signatures